acquire/acquire.py

```diff
@@ -852,7 +863,6 @@ class Misc(Module):
         ("dir", "sysvol/windows/system32/sru"),
         ("dir", "sysvol/windows/system32/drivers/etc"),
         ("dir", "sysvol/Windows/System32/WDI/LogFiles/StartupInfo"),
-        ("dir", "sysvol/windows/sysvol/domain/policies/"),
```

Comment:
I assume this path will now come from `HKLM\\SYSTEM\\CurrentControlSet\\Services\\Netlogon\\Parameters`?